.ci/openvino-onnx/Jenkinsfile

```diff
@@ -2,6 +2,9 @@
 // Copyright (C) 2018-2020 Intel Corporation
 // SPDX-License-Identifier: Apache-2.0
 
+DOCKER_HOME = "/openvino"
```

Comment:
This is not a good variable name.

Reply:
Maybe `START_DIR`?

Reply:
I resolved this by removing this variable ;)